testing/src/main/graphql/betterbotz/aggregations/GetOrdersAllFunctions.graphql

```diff
@@ -0,0 +1,23 @@
+query GetOrdersAllFunctions($filter: OrdersFilterInput) {
+    Orders(filter: $filter) {
+        pageState
+        values {
+            id
+            prodId
+            prodName
+            customerName
+            address
+            description
+            price
+            sellPrice
+            _int_function(name: "max", args: ["value_int"])
```

Comment:
Related to my comment above, could these be `function(name: "max", args: ["value_int"])` or `numeric_function(name: "max", args: ["value_int"])`?